docs/upgrade-guide.md

```diff
@@ -15,18 +15,18 @@ luma.gl largely follows [SEMVER](https://semver.org) conventions. Breaking chang
 v9.2 brings full WebGPU support. Some additional deprecations and breaking changes have been necessary, but apart from the `Texture` -> `AsyncTexture` split, impact on most applications should be minimal. 
 
 **Major change: Texture and AsyncTextures**
-- `device.createTexture()` no longer accepts `props.mipmaps`: Use `AsyncTexture` or call texture.generateMipmapsWebGL()`
-- `device.createTexture()` no longer accepts `props.mipmaps`: Use `AsyncTexture` or call `texture.generateMipmapsWebGL()`
+- The `Texture` class has been simplified to the minimum API required for GPU portability. The  `AsyncTexture` texture class provides a higher-level API and is recommended for most applications.
+- `device.createTexture()` no longer accepts `props.data`: Use `AsyncTexture` or call `texture.setImageData()`
+- `device.createTexture()` no longer accepts `props.mipmaps`: Use `AsyncTexture` (or call `texture.generateMipmapsWebGL()`)
 
 **Major change: Removal of WebGL uniform support**
-- `Model.setUniformsWebGL()` dropped, use uniform buffer bindings
-- `Renderpipeline.setUniformsWebGL()` dropped, use uniform buffer bindings
+- The transition from uniforms to uniform buffers is complete, and remaining support for non-buffer uniforms has been removed.
+- `core`: `Renderpipeline.setUniformsWebGL()` dropped, use uniform buffer bindings
+- `engine`: `Model.setUniformsWebGL()` dropped, use uniform buffer bindings
```

Comment:
This is pretty aggressive, are we sure we want to remove it this version?

Reply:
- No, this is on `master` which is targeting v9.2. 
-v9.1 is now on `9.1-release` branch.